### ISSUE TITLE
Fix failing accessibility tests

### ISSUE THREAD

**Issue by aberikashvili** [SAP/fundamental-styles#2388]
**Accessibility tests failing**
For the following list of components, there're failing accessibility test which need to be fixed:

> avatar
    object-list
    object-status
    tabs
    facets
    counter
    busy-indicator
    object-marker
    multi-input
    token
    notification
    card
    list-grid
    wizard
    table
    numeric-content
    shellbar
    user-menu
    dialog
    avatar-group
    side-navigation
    multi-combo-box
    generic-tile
    dynamic-page

**To Reproduce**
1. Open Terminal / CMD
2. Navigate to `fundamental-styles` repository
3. Remove `package-lock.json`
4. Remove `node_modules`
5. Run `npm install`
6. Run `npm test`
7. See failing accessibility tests

**Expected behavior**
`npm test` should successfully pass
